### PR TITLE
 Make options easily reachable in mobile view

### LINKED ISSUE #80: Hard to use on mobile
You have to scroll to top to access the options and search bar which is a bit cumbersome on small screen devices.

### PR DESCRIPTION
**Pull request by gpalsingh**
Moved checkboxes to a slider menu.
Header stays fixed on top making it easier to search.
Typing in search box automatically scrolls to top.
"Back to top" works correctly on all screen sizes.

Closes #80

### CODE REVIEW

**Comment by gpalsingh** [comment]
Usage example: https://gyazo.com/3d4f5e09079e06e8e9b0bda04ff817fb
Live Demo: https://hackerqtest.herokuapp.com/

**Comment by frankcash** [comment]
Apparently this breaks the travis build, could you fix?

File: public/app.js [hunk 1]
```diff
@@ -68,8 +68,15 @@ app.controller("AppTest", function($scope, $http, $location, $anchorScroll){
...
-    $location.hash('refresh');
-    $anchorScroll();
+    $anchorScroll.yOffset = document.getElementById('container').offsetTop;
+    $anchorScroll('container');
+  };
+
+  $scope.optionsCloseVisible = false;
+  $scope.toggleOptionsMenu = function(){
+    const options = document.getElementById('news-options');
+    options.style.width = options.style.width ? '' : '100%';
```
Comment (frankcash) [inline public/app.js]: please try to use if statements instead of ternary operators

File: public/app.js [hunk 2]
```diff
@@ -68,8 +68,15 @@ app.controller("AppTest", function($scope, $http, $location, $anchorScroll){
...
 
   $scope.goToTop = function(){
-    $location.hash('refresh');
-    $anchorScroll();
+    $anchorScroll.yOffset = document.getElementById('container').offsetTop;
+    $anchorScroll('container');
+  };
+
+  $scope.optionsCloseVisible = false;
+  $scope.toggleOptionsMenu = function(){
```
Comment (frankcash) [inline public/app.js]: could you do `()=>{}` notation for a function?

File: public/app.js (hunk 1 above)
Comment (gpalsingh) [reply public/app.js]: Added in 4efa46b

File: public/app.js (hunk 2 above)
Comment (gpalsingh) [reply public/app.js]: Added in cc45d01

**Comment by frankcash** [comment]
Hello,  I merged this into master and it seemed to break things.  I can investigate more later on, I'm sorry.

File: public/app.js [hunk 3]
```diff
@@ -67,9 +67,20 @@ app.controller("AppTest", function($scope, $http, $location, $anchorScroll){
     });
   };
 
-  $scope.goToTop = function(){
-    $location.hash('refresh');
-    $anchorScroll();
+  $scope.goToTop = () => {
+    $anchorScroll.yOffset = document.getElementById('container').offsetTop;
```
Comment (frankcash) [inline public/app.js]: why did you remove the location.hash part?

Comment (gpalsingh) [reply public/app.js]: I couldn't get `$anchorScroll.yOffset` to work with location.hash without changing `yOffset` globally so I used this instead.

**Comment by gpalsingh** [comment]
What issues did you face? I tested it on chrome and it worked fine.
